.gitlab-ci.yml

```
@@ -110,99 +104,101 @@ test:acceptancetests:
+  stage: .post
   tags:
     - hetzner-amd-beefy
   rules:
-    - if: $CI_COMMIT_BRANCH =~ "/^\d+\.\d+\.x$/" && $RUN_RELEASE == "true"
-    - if: $CI_COMMIT_BRANCH == $DEFAULT_BRANCH && $RUN_RELEASE == "true"
+    # Only make available for protected branches (main and maintenance branches)
+    - if: '$CI_COMMIT_TAG =~ /^v\d+\.\d+\.\d+$/'
+      when: never
+    - if: $CI_COMMIT_BRANCH =~ "/^v\d+\.\d+\.x$/" && $CI_COMMIT_BRANCH == $DEFAULT_BRANCH
+      when: manual
+      allow_failure: true
+    # Exclude release tags
+    - if: $CI_COMMIT_TAG
+      when: never
+  needs:
+    - job: changelog
   script:
-    - npm install -g release-please
     - release-please github-release
-        --token=${GITHUB_BOT_TOKEN_REPO_FULL}
-        --repo-url=${GITHUB_REPO_URL}
-        --target-branch=${CI_COMMIT_REF_NAME}
+      --token=${GITHUB_BOT_TOKEN_REPO_FULL}
+      --repo-url=${GITHUB_REPO_URL}
+      --target-branch=${CI_COMMIT_REF_NAME}
 
 release:mender-docs-changelog:
-  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/alpine:latest
-  stage: release
+  image: "registry.gitlab.com/northern.tech/mender/mender-test-containers:release-please-v1-master"
+  stage: .post
   tags:
     - hetzner-amd-beefy
   rules:
-    - if: $CI_COMMIT_TAG =~ "/^\d+\.\d+\.\d+$/"
+    # Only make available for stable branches
+    - if: '$CI_COMMIT_TAG =~ /^v\d+\.\d+\.\d+$/'
+      allow_failure: true
   before_script:
-    - apk add git bash curl jq github-cli
-    - |
-      echo "INFO - setting up git"
-      git config --global user.email ${GITHUB_USER_EMAIL}
-      git config --global user.name ${GITHUB_USER_NAME}
-      export GITHUB_TOKEN=${GITHUB_BOT_TOKEN_REPO_FULL}  # used for gh cli
+    # Setting up git
+    - git config --global user.email "${GITHUB_USER_EMAIL}"
+    - git config --global user.name "${GITHUB_USER_NAME}"
+    # GITHUB_TOKEN for Github cli authentication
+    - export GITHUB_TOKEN=${GITHUB_CLI_TOKEN}
   script:
-    - git clone https://${GITHUB_USER}:${GITHUB_TOKEN_NT}@github.com/${GITHUB_CHANGELOG_REPO_URL}
+    - git clone https://${GITHUB_USER_NAME}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/${GITHUB_CHANGELOG_REPO_URL}
     - cd ${GITHUB_CHANGELOG_REPO_URL#*/}
-    - git checkout -b changelog-${CI_PIPELINE_ID}
-    - |
-      cat ../docs_header.md > ${CHANGELOG_REMOTE_FILE}
-      cat ../CHANGELOG.md | grep -v -E '^---' >> ${CHANGELOG_REMOTE_FILE}
-      git add ${CHANGELOG_REMOTE_FILE}
-      git commit -s -m "chore: add app-update-module changelog"
-      git push origin changelog-${CI_PIPELINE_ID}
-    - gh pr create --title "Update CHANGELOG.md" --body "Automated change to the CHANGELOG.md file" --base master --head changelog-${CI_PIPELINE_ID}
+    - git checkout -b changelog-${CI_JOB_ID}
+    - cat ../.docs_header.md > ${CHANGELOG_REMOTE_FILE}
+    - cat ../CHANGELOG.md | grep -v -E '^---' >> ${CHANGELOG_REMOTE_FILE}
+    - git add ${CHANGELOG_REMOTE_FILE}
+    - git commit -s -m "chore: add $CI_PROJECT_NAME changelog"
```

```suggestion
    - 'git commit -s -m "chore: add $CI_PROJECT_NAME changelog"'
```

:point_up: Quote it to fix the YAML syntax